.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2.1
+
+executors:
+  default:
+    docker:
+      - image: circleci/golang:1.12
+
+jobs:
+  build:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - run:
+          name: Setup Environment Variables
+          command: |
+            echo "export PATH=$GOPATH/bin:$PATH" >> $BASH_ENV
+            echo "export PATH=$PATH:$PWD/bin" >> $BASH_ENV
+            echo "HUGO_VERSION='0.58.3'" >> $BASH_ENV
+            echo "MATTERMOST_SERVER_RELEASE_TAG=v5.17.1" >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: install Node JS
+          command: |
+            sudo apt-get update
```

Comment:
you dont need to install node you can use the circleci image with node `circleci/golang:1.12-node`

Reply:
Thanks, I overlooked this image. Fixed in the https://github.com/mattermost/mattermost-developer-documentation/pull/450/commits/d5fb2f2f062a2545cb4c8a43245f40b816b157c0